### PR TITLE
COTR-33 Connect player signals to hud

### PR DESCRIPTION
**Pull request by Elijuszek**
Connected ChangedHealth signal
    Connected ChangedMaxHealth signal
    ConnectedChangedAmmo signal
    Connected ChangedMaxAmmo signal
    Connected ChangedGold
    Connected ChangedXp

### CODE REVIEW

File: Scenes/UI/HUD/GUI.cs [hunk 1]
```diff
@@ -3,9 +3,8 @@
 /// <summary>
 /// GUI class that contains graphical UI elements and methods to change them
 /// </summary>
-public partial class GUI : CanvasLayer
+public partial class GUI : Control
 {
-    /// <summary>
     /// Class parameters
     /// </summary>
     private ProgressBar healthBar;
```
Comment (Karolinskis) [inline Scenes/UI/HUD/GUI.cs:10]: Fix this summary

**Review by Karolinskis — CHANGES_REQUESTED** [review]
Fix please 😄

File: Scenes/Maps/Main/Map.cs [hunk 2]
```diff
@@ -1,14 +1,30 @@
...
+
+/// <summary>
+/// Class for implementing enviroment
+/// </summary>
 public partial class Map : Node2D
 {
-    Player player;
-    PackedScene playerScene;
+    private Player player;
+    private PackedScene playerScene;
```
[the remark this answers is not in the document]
Comment (Elijuszek) [reply Scenes/Maps/Main/Map.cs:9]: Done

File: Scenes/Actors/Player/Player.cs [hunk 3]
```diff
@@ -25,12 +24,15 @@ public partial class Player : Actor
     private Vector2 movementDirection = Vector2.Zero;
```
[the remark this answers is not in the document]
Comment (Elijuszek) [reply Scenes/Actors/Player/Player.cs:24]: Done

File: Scenes/Actors/Player/Player.cs [hunk 4]
```diff
@@ -25,12 +24,15 @@ public partial class Player : Actor
...
     private Vector2 attackDirection = Vector2.Zero;
 
+    private LevelSystem levelSystem;
+
     // TODO: lacking damagePopup scene implementation
     //private PackedScene damagePopup = (PackedScene)ResourceLoader.Load("res://Scenes/UI/Popups/DamagePopup.tscn");
     //private GUI gui;
 
     private Globals globals;
```
[the remark this answers is not in the document]
Comment (Elijuszek) [reply Scenes/Actors/Player/Player.cs:33]: Done

File: Scenes/UI/HUD/GUI.cs (hunk 1 above)
Comment (Elijuszek) [reply Scenes/UI/HUD/GUI.cs:10]: Done

**Review by Karolinskis — APPROVED** [review]

**Pull request merged**